### DISCUSSION TITLE
エラー

### DISCUSSION THREAD

**Discussion by 11EDA** [aclap-dev/video-downloadhelper#545] [Help and Q&A]
### Your question

ダウンロードできない

### Addon & CoApp version and details

{
  "version": "9.0.1.13",
  "target": "google",
  "channel": "stable",
  "page_url": "https://missav.com/ja/fc2-ppv-4436963",
  "error": {
    "details": "SideDownload error: : [in#0 @ 0000012856a8b800] Error opening input: Server returned 403 Forbidden (access denied)\r\nError opening input file https://surrit.com/ca6b67b3-e930-4606-97a7-f66be3423013/1080p/video.m3u8.\r\nError opening input files: Server returned 403 Forbidden (access denied)\r\n",
    "downloadable_id": "downloadable_8727870302938342",
    "error": "coapp_failure",
    "id": "coapp_failure_6fa5d89c-1b31-4840-9218-701c34617b9e"
  }
}

**Comment by GrampaWildWilly** [comment]
Read #519.
